geiger/src/lib.rs

```diff
@@ -102,6 +102,12 @@ impl Add for CounterBlock {
     }
 }
 
+pub enum DetectionStatus {
+    NoneDetectedForbidsUnsafe,
+    NoneDetectedAllowsUnsafe,
+    UnsafeDetected,
+}
+
```

Comment:
The name of this type should probably be renamed to something like `CrateDetectionStatus` and moved back to the `cargo-geiger` crate since the `geiger` library doesn't know about crates, only files.